src/main/java/org/schemaspy/view/WriteStats.java

```diff
@@ -27,22 +27,17 @@
  * @author John Currier
  */
 public class WriteStats {
```

Comment:
We might as well rename the class now, which allows us to (re)use it more generally. It's a `Tally`, maybe?

Reply:
Used LongAdder jdk class